A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/engines.cfg

```diff
@@ -38,13 +38,13 @@ oil_temp_cooling_constant = 0.21
 oil_temp_heating_constant = 700
 oil_temp_tc = 0.03
 oil_temp_tuning_constant = 1
-oil_press_max = 8640.2
+oil_press_max = 18640.2

```

Comment:
18640.2 Pounds per sq foot is 130PSI thats supposed to be the gauge max as attached in the image with the PR.